src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -190,132 +193,118 @@
     }
 };
 
-function calc(operation, sample, scenario) {
-    let data = iterations(scenario, Phase.MEASURE).map(iteration => iteration.values[sample.name]);
-    return OPERATIONS[operation](data);
-}
-
-var app = new Vue({
-    el: '#app',
-    data: benchmarkResult,
-    filters: {
-        "numeric": (value) => value.toFixed(2),
-        "percent": (value) => (value * 100).toFixed(2) + "%"
-    }
-});
-
-const COLORS = ["#527AB3", "#56ac76", "#f44336", "#d49c3e", "#211f2d"];
-class PaletteGenerator {
-    constructor() {
-        this.colors = COLORS.slice();
-    }
-    nextColor() {
-        if (this.colors.length) {
-            return this.colors.shift();
-        } else {
-            return "#8B899A";
-        }
-    }
-};
-
 function convertScenarioToChart(scenario, sample, sorted, color) {
     var data = iterations(scenario, Phase.MEASURE)
-            .map(iteration => iteration.values[sample]);
+            .map(iteration => iteration.values[sample.name]);
     if (sorted) {
         data = data.sort((a, b) => b - a);
     }
     return {
-        label: `${scenario.definition.title} (${sample}) ${scenario.definition.buildTool}`,
+        label: sample.name,
         showLine: true,
         steppedLine: "middle",
         pointRadius: 0,
         pointHitRadius: 10,
         hoverRadius: 6,
         fill: false,
-        borderWidth: 3,
+        borderWidth: sample.thickness,
         pointBackgroundColor: color,
         borderColor: color,
         data: data
     };
 }
 
-function convertData(samples, sorted) {
-    let palette = new PaletteGenerator();
-    return samples.map(sample => benchmarkResult.scenarios.map(scenario => convertScenarioToChart(scenario, sample, sorted, palette.nextColor()))).flat();
-}
-
-function updateData() {
-    let sorted = $("input[name='sorting']:checked").val() === "sorted";
-    let samples = $("#sample").val();
-    chart.data.datasets = convertData(samples, sorted);
-    chart.update();
-}
-
-let ctx = document.getElementById('samples').getContext('2d');
-let maxMeasuredIterations = Math.max(...(benchmarkResult.scenarios.map(scenario => iterations(scenario, Phase.MEASURE).length)));
-let chart = new Chart(ctx, {
-    type: 'line',
+var app = new Vue({
```

Comment:
```suggestion
const app = new Vue({
```

No need to leave this mutable.

Reply:
I've fixed this and others.